include/fastdds/dds/rpc/RPCTypeObjectSupport.hpp

```diff
@@ -0,0 +1,139 @@
+// Copyright 2025 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file RPCTypeObjectSupport.hpp
+ */
+
+#ifndef FASTDDS_DDS_RPC__RPCTYPEOBJECTSUPPORT_HPP
+#define FASTDDS_DDS_RPC__RPCTYPEOBJECTSUPPORT_HPP
+
+#include <fastdds/dds/xtypes/type_representation/TypeObject.hpp>
+#include <fastdds/fastdds_dll.hpp>
+
+namespace eprosima {
+namespace fastdds {
+namespace dds {
+namespace rpc {
+
+/**
+ * @brief Register RpcException related TypeIdentifier.
+ *        Fully-descriptive TypeIdentifiers are directly registered.
+ *        Hash TypeIdentifiers require to fill the TypeObject information and hash it, consequently, the TypeObject is
+ *        indirectly registered as well.
+ *
+ * @param[out] TypeIdentifier of the registered type.
```

Comment:
Docstring causes a Doxygen error. Apply on each method:
```suggestion
 * @param[out] type_ids TypeIdentifier of the registered type.
```